catalog.html

```diff
@@ -16,96 +16,110 @@
+        <div class="sort-wrapper">
+          <h3 class="main-catalog-title">Найдено: 3</h3>
+          <span class="main-catalog-subtitle">Сортировка:</span>
+        <ul class="sort-list">
+          <li class="sort-item"><a href="#" class="sort-item-link price">По цене</a></li>
+          <li class="sort-item"><a href="#" class="sort-item-link">По типу</a></li>
+          <li class="sort-item"><a href="#" class="sort-item-link">По рейтингу</a></li>
+        </ul>
+        <div class="arrow-wrapper">
+          <a href="#" class="arrow"><img src="img/arrow-up.svg" alt="стрелка вверх"></a>
+          <a href="#" class="arrow"><img src="img/arrow-down.svg" alt="стрелка вниз"></a>
         </div>
-        <ul class="items hotels-list">
-          <li class="hotel-item">
-            <article class="hotel-card">
-              <img src="img/amara-resort-spa.jpg" alt="Гостиница в Седоне" width="135" height="90" class="hotel-card-image">
-              <div class="hotel-card-wrapper">
-                <h3 class="hotel-card-title"><a href="#">Amara Resort & Spa</a></h3>
-                <span>Гостиница</span>
-                <span>от 4000 Р.</span>
-                <div class="button-wrapper">
-                  <a href="#">Подробнее</a>
-                  <a href="#">Забронировать</a>
-                </div>
-               </div>
-            </article>
-          </li>
-          <li class="hotel-item">
-            <article class="hotel-card">
-              <img src="img/desert-qualin.jpg" alt="Мотель в Седоне" width="135" height="90" class="hotel-card-image">
-              <div class="hotel-card-wrapper">
-                <h3 class="hotel-card-title"><a href="#">Desert Quail Inn</a></h3>
-                <span>Мотель</span>
-                <span>от 3000 Р.</span>
-                <div class="button-wrapper">
-                  <a href="#">Подробнее</a>
-                  <a href="#">Забронировать</a>
-                </div>
-              </div>
-            </article>
-          </li>
-          <li class="hotel-item">
-            <article class="hotel-card">
-              <img src="img/villas-at-poco-diablo.jpg" alt="Апартаменты в Седоне" width="135" height="90" class="hotel-card-image">
-              <div class="hotel-card-wrapper">
-                <h3 class="hotel-card-title"><a href="#">Villas at Poco Diablo</a></h3>
-                <span>Апартаменты</span>
-                <span>от 2000 Р.</span>
-                <div class="button-wrapper">
-                  <a href="#">Подробнее</a>
-                  <a href="#">Забронировать</a>
-                </div>
+         </div>
+        
+         <section class="hotel-card-wrapper">
+          <article class="hotel-card">
+            <img src="img/amara-resort-spa.jpg" alt="фотография гостиницы" class="hotel-image" width="135" height="90" class="hotel-image">
+            <div class="hotel-wrapper">
+              <h3 class="hotel-title">Amara Resort & Spa</h3>
```

Comment:
Почему ссылку убрал? 
```suggestion
              <h3 class="hotel-title"><a href="#">Amara Resort & Spa</a></h3>
```